### PR TITLE
Dust off the `README.md` and pick some other nits

### PR DESCRIPTION
**Pull request by webbnh**
This PR is an accumulation of random things which I've spotted while doing code reviews and investigating other changes:
- the top-level `README.md` needed updating
- a couple of files were suffering from cut-and-pastage
- the `docs/.gitignore` file was missing its trailing newline.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -158,18 +152,23 @@ a sub-directory name found in `agent/bench-scripts/tests`. For example:
 The first runs all the `pbench-fio` tests, while the second runs all the
 `pbench-uperf` and `pbench-linpack` tests.
 
+You can run the `build.sh` script to execute the linters, to run the unit tests
```
Comment (dbutenhof) [inline README.md:155]: Dang GitHub won't let me comment where I'd like: but some whim caused me to expand above this diff and I found several `tox` references to the server `legacy` tests up there. If you feel like it, you might nuke them while you're here.

File: README.md [hunk 2]
```diff
@@ -194,14 +193,13 @@ starting with the `v0.70.0` release (`v<Major>.<Minor>.<Release>[-<Build>]`).
 Prior to the v0.70.0 release, the scheme used was mostly `v<Major>.<Minor>`,
 where we only had minor releases (`Major = 0`).
 
-The practice of using `-agent` or `-server` is also ending with the `v0.70.0`
-release.
+The practice of using `-agent` or `-server` ended with the `v0.70.0` release.
```
Comment (dbutenhof) [inline README.md:196]: 0.70 is ancient history: should we just nuke this whole confusingly historical section while we're sweeping?

File: README.md [hunk 3]
```diff
@@ -9,23 +9,22 @@ for those systems, and specified telemetry or data from various tools (`sar`,
 
 The second sub-system is the `pbench-server`, which is responsible for
 archiving result tar balls, indexing them, and unpacking them for display.
+It provides a RESTful API which can be used by client applications, such as
+the Pbench Dashboard, which provides a web-based interface to the Pbench
+Server.
```
Comment (dbutenhof) [inline README.md:14]: Although the "dashboard" stuff probably doesn't belong here, I'd have preferred to keep the "RESTful API" reference. Also, technically, the "unpacking them for display" is a bit outdated, and maybe a more general "managing access to components/elements/files/inventory within them?"???

File: README.md (hunk 2 above)
Comment (webbnh) [reply README.md:196]: I removed the changed line here; I decided to leave the verbiage about versions (GitHub still has nearly 60 back-versions).

File: README.md (hunk 3 above)
Comment (webbnh) [reply README.md]: Let me know if the result needs...more.

File: README.md [hunk 4]
```diff
@@ -8,24 +8,24 @@ for those systems, and specified telemetry or data from various tools (`sar`,
 `vmstat`, `perf`, etc.).
 
 The second sub-system is the `pbench-server`, which is responsible for
-archiving result tar balls, indexing them, and unpacking them for display.
+archiving result tar balls, indexing them, and managing access to their
+contents.
+It provides a RESTful API which can be used by client applications, such as
+the Pbench Dashboard, which provides a web-based interface to the Pbench
+Server.
```
Comment (dbutenhof) [inline README.md:15]: Trying again: the RESTful API stuff is good, having the dashboard reference in the middle of the sentence is distracting and isn't needed here. Ideally, "Pbench Dashboard" is a distinct subsystem deserving of its own paragraph, and if you want to mention that it's a RESTful API client, that's fine.

(And really, though it's outside of where GitHub lets me comment, this "web-server" subsystem is a total crock. The server packages a few `npm` modules and the `jschart.js` script to support displays, but there's nothing here in any way resembling a "web server" ... we're just stocking some browser snacks, and it's all part of the Pbench Server package. With Quisby, maybe nobody will even care anymore. 🤞🏻 )

**Review by dbutenhof — DISMISSED** [review]

File: README.md [hunk 5]
```diff
@@ -215,13 +209,9 @@ we build, with the following exceptions for tag names:
...
 the functional tests, and other verification and testing environments use
-container images from public repositories and non-public ones.  The CI jobs
+container images from remote image registries.  The CI jobs
 obtain references to those repositories using Jenkins credentials.  When a
-developer runs those same jobs locally, you can create two files with the
-appropriate contents locally:
-
-  * `${HOME}/.config/pbench/ci_registry.name`
-  * `${HOME}/.config/pbench/public_registry.name`
+running those same jobs locally, you can provide the registry via
```
Comment (ndokos) [inline README.md:214]: Delete the "a"  at the end of l.213

**Review by ndokos — APPROVED** [review]
One small nit if you care to fix it.

File: README.md (hunk 4 above)
Comment (dbutenhof) [reply README.md]: Still don't like the dashboard phrase and I think it complicates without helping any; but the overall paragraph is improved somewhat. I give up.

**Review by dbutenhof — APPROVED** [review]

**Review by ndokos — APPROVED** [review]

**Pull request merged**